config/default/app_settings.json

```diff
@@ -353,9 +353,9 @@
     "targets": {
```

Comment:
Can you tell me what changed in this file? This config is used as the configuration when the app is first started so we have to be a little careful changing it, and it's hard to diff because it's minified...

Reply:
Nothing should have changed. Must be formatting changes i will revert.